lib/STREAMS.md

```diff
@@ -132,3 +132,19 @@ Can return synchronously. Use `await` or wrap the return value in `Promise.resol
 ### readStream.peekBuffer(byteCount)
 
 Like `readStream.read` and `readStream.peek`, but returns a Buffer instead of a string.
+
+### readStream.peeker([encoding])
```

Comment:
Is `peeker` ever necessary? I can't imagine any use-case where you'd want `peek` to be a stream.

Reply:
If one of the `read` methods were to be used within the loop, I think it could be useful for things like binary streams. I can't think of a case offhand where it would be useful for text, though.

Reply:
Hm, I guess I can envision something like that. I still think using a stream for that kind of use-case is really unreadable, especially since it's not just streaming the entire data, but only the parts you don't read. I think a while-loop would make it much more readable so I'd rather enforce that by not supporting a peek stream.